src/wpool_pool.erl

```diff
@@ -158,6 +158,28 @@ broadcast(Name, Cast) ->
     lists:foreach(fun(Worker) -> ok = wpool_process:cast(Worker, Cast) end,
                   all_workers(Name)).
 
+%% @doc Calls all workers in the pool in parallel
+-spec broadcall(wpool:name(), term(), timeout()) ->
+                   {[Replies :: term()], [Errors :: term()]}.
+broadcall(Name, Call, Timeout) ->
+    Workers = all_workers(Name),
+    ReqId0 = gen_server:reqids_new(),
+    RequestFold = fun(Worker, Acc) -> gen_server:send_request(Worker, Call, Name, Acc) end,
+    ReqId1 = lists:foldl(RequestFold, ReqId0, Workers),
+    WaitFold =
+        fun(_, {Coll, Replies, Errors}) ->
+           case gen_server:receive_response(Coll, Timeout, true) of
+               {{reply, Reply}, _, Coll1} ->
+                   {Coll1, [Reply | Replies], Errors};
+               {{error, Error}, _, Coll1} ->
+                   {Coll1, Replies, [Error | Errors]};
+               timeout ->
+                   exit({timeout, {?MODULE, broadcall, [Name, Call, Timeout]}})
```

Comment:
I understand that you're assuming here that if one server timed out, then every server after this one will time out as well since all share the same `Timeout` value. Is that correct?
If that's true, a nice comment stating that would be helpful.

Reply:
Done 👌🏽